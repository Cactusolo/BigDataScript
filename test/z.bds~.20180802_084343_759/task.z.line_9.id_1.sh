#!/bin/bash -e

cd '/Users/kqrw311/workspace/BigDataScript/test'

# SYS command. line 9
./test/failOnce.pl
# Checksum: 44c0dde0
