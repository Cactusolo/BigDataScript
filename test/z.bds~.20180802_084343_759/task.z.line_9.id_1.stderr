/Users/kqrw311/workspace/BigDataScript/test/z.bds~.20180802_084343_759/task.z.line_9.id_1.sh: line 6: ./test/failOnce.pl: No such file or directory
